modules/apply-generated-policies.adoc

```diff
@@ -0,0 +1,19 @@
+// Module included in the following assemblies:
+//
+// * operating/manage-network-policies.adoc
+:_module-type: PROCEDURE
+[id="apply-generated-policies_{context}"]
+= Applying generated policies
+
+[role="_abstract"]
+
+You can apply the generated network policies from the RHACS portal.
+
+.Procedure
+* To directly apply the generated policies in the cluster from within {product-title}, select *Apply Network Policies*.
+
+[WARNING]
+====
+Directly applying network policies may cause problems for running applications.
+Always download and test the network policies in development environment or testing clusters, before you apply them to production workloads.
```

Comment:
* in development environment -> in a development environment ?
* testing -> test ?
* And comma could be removed